pcap/search.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 var (
-	// ErrNoPacketsFound is an error indicating no packets have been found.
-	ErrNoPacketsFound = zqe.E(zqe.NotFound, "no packets found")
+	// ErrNoPcapsFound is an error indicating no packets have been found.
+	ErrNoPcapsFound = zqe.E(zqe.NotFound, "no pcaps found")
```

Comment:
"no pcaps found" sounds like we haven't found any packet captures, as opposed to having found packet captures but no matching packets inside them. Is this what we want?

Reply:
So what I hear you proposing is the error message should remain: "no packets found"?

Reply:
I don't know the the API. 

If the error is that there is no pcap, then no pcaps found. If the error is "no matching packets inside the pcaps", then yes "no packets found" (or "no matching packets found", if that makes sense).

Reply:
Yeah I agree with the "no packets found" here.